Scripts/script-ReadFile.yml

```
@@ -10,7 +10,14 @@ script: |-
       maxFileSize = 1024**2
   res = demisto.executeCommand('getFilePath', {'id': demisto.args()['entryID']})
   with open(res[0]['Contents']['path'], 'r') as f:
-      data = f.read(maxFileSize).decode('unicode_escape').encode('utf-8')
+      data = f.read(maxFileSize)
+      try:
+          data = data.decode('unicode_escape')
+      except UnicodeDecodeError:
+          pass  # Passing in a case we have \U or \N in the file
+
+      data = data.encode(demisto.args().get("encoding"))
```

data returned to the context should be utf-8 encoded always
```suggestion
      data = data.encode('utf-8')
```